elasticsearch/src/test/java/org/apache/calcite/adapter/elasticsearch/EmbeddedElasticsearchNode.java

```diff
@@ -97,8 +97,8 @@ public static synchronized EmbeddedElasticsearchNode create() {
         .put("path.data", data.getAbsolutePath())
         .put("http.type", "netty4")
         // allow multiple instances to run in parallel
-        .put("transport.tcp.port", 0)
-        .put("http.port", 0)
+        .put("transport.tcp.port", 9300)
+        .put("http.port", 9200)
```

Comment:
Why make this change? Back it out.

Reply:
I will discard this changes,thanks a lot.